ga4gh/cli.py

```diff
@@ -638,7 +641,10 @@ def addClientGlobalOptions(parser):
     parser.add_argument(
         "--workarounds", "-w", default=None, help="The workarounds to use")
     parser.add_argument(
-        "--key", "-k", help="The auth key to use")
+        "--key", "-k",
+        help=(
+         "The auth key to use. Browse to https://<server>/_login to get one."),
+        required=True)
```

Comment:
OIDC mandates https, so I'll make it default to on if it's oidc. Otherwise yes i'll add a boolean argument.